packages/common/src/intl/locales/en.json

```diff
@@ -1,7 +1,7 @@
 {
   "app.admin": "Admin",
   "app.customer_invoices": "Customer Invoices",
-  "app.customer-invoice": "Customer Invoices",
+  "app.customer-invoice": "Invoices",

```

Comment:
Uhh.. it looks better 😁

Reply:
I've also notice that there's a `-` and a `_` 🙄  oops.
And there's a plan to rename these to `Outgoing shipment` and `Incoming shipment` 😬